src/tables.ts

```diff
@@ -116,3 +116,14 @@ export const oauthTokens = new DatabaseTable<{
 	id: string;
 	time: number;
 }>(psdb, 'oauth_tokens', 'id');
+
+export const teams = new PGDatabase<{
+	teamid: string;
+	team: string;
+	ownerid: string;
+	format: string;
+	title: string | null;
+	date: Date;
+	private: boolean;
+	views: number;
+}>(Config.teamsdb);
```

Comment:
I dislike hard-wiring the connection details to a global variable. I've explained this position at length with the  MySQL code

Reply:
I changed this to a general configuration for postgres connection details, but I'm not sure what you want me to do other than that. `Config` isn't currently a global.